test/LEGO.AsyncAPI.Tests/Bindings/Sns/SnsBindings_Should.cs

```diff
@@ -167,20 +167,18 @@ public void SnsOperationBinding_WithFilledObject_SerializesAndDeserializes()
           x-identifierExtension:
             identifierXPropertyName: identifierXPropertyValue
         filterPolicy:
-          attributes:
-            store:
-              - asyncapi_corp
-            contact: dec.kolakowski
-            event:
-              - anything-but: order_cancelled
-            order_key:
-              transient: by_area
-            customer_interests:
-              - rugby
-              - football
-              - baseball
-          x-filterPolicyExtension:
-            filterPolicyXPropertyName: filterPolicyXPropertyValue
+          store:
+            - asyncapi_corp
+          contact: dec.kolakowski
+          event:
+            - anything-but: order_cancelled
+          order_key:
+            transient: by_area
+          customer_interests:
+            - rugby
+            - football
+            - baseball
+        filterPolicyScope: MessageAttributes
```

Comment:
looks like the extension was removed.
Lets add an extension to test that the binding is extensible

Reply:
I removed this extension as the `attributes` property no longer exists and the filter policy lives directly under the object now. So I thought the extension for the filter policy object should now be covered by the general consumer extension (https://github.com/LEGO/AsyncAPI.NET/pull/128/files#diff-0b77af8e82f20917a60bde7519aebfcb0c7f67315a86a89636c04ba6b16fd315L204)?

Reply:
You are absolutely right. I totally missed that!